stable/influxdb/templates/service.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+  {{- if .Values.config.http.enabled }}
+  - name: api
+    port: {{ .Values.config.http.bind_address }}
```

Comment:
This is a little nitpicky on my part but these are technically bind_ports rather then bind_addresses